.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Bug report
+about: Unexpected behavior? Tell us more
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+# Describe your issue
+
+(Tell us what the problem is in a few words)  
+*Ex: As an end user, when I access the console and see a service inventory, if attributes have too long values, I can't see any of the right buttons anymore*
+
+# Steps to reproduce
+
+1. step 1
+2. step 2
+3. ...
+4. result
+
+# Expected behavior
+
+(Tell us what you expected this list of steps to do)  
+*Ex: On the right side of the service I should see 4 buttons: show resources, edit, delete and diagnose*
+
+# Work around (if any)
+
+(You can add detail about the resolution of this issue here)  
+*Ex: none*
+
+# Your environment
+
+Please provide the following information:
+ - Current Inmanta version: 
+ - Operating System: 
+ - Python version:
+ - PostgreSQL version:
+
+Using Linux? You can replace the list by the output of the following command:  
```

Comment:
Is this useful? Because if you are using a virtualenv and have installed pg10 from redhat software collections half of the commands will not work.